pkg/apis/pipeline/v1/pipeline_types_test.go

```diff
@@ -606,6 +606,21 @@ func TestPipelineTask_validateMatrix(t *testing.T) {
 			}},
 		},
 		wantErrs: apis.ErrMultipleOneOf("matrix[foobar]", "params[foobar]"),
+	}, {
+		name: "duplicate parameters in matrix params",
```

Comment:
to be consistent with the other test, and make it clear which params we're referring to
```suggestion
		name: "duplicate parameters in matrix.params",
```